src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services_inline_requires.php

```diff
@@ -37,10 +37,10 @@ public function __construct()
         $this->aliases = [];
 
         $this->privates['service_container'] = function () {
-            include_once $this->targetDirs[1].'/includes/HotPath/I1.php';
-            include_once $this->targetDirs[1].'/includes/HotPath/P1.php';
-            include_once $this->targetDirs[1].'/includes/HotPath/T1.php';
-            include_once $this->targetDirs[1].'/includes/HotPath/C1.php';
+            include_once \dirname(__DIR__, 0 + 1).'/includes/HotPath/I1.php';
```

Comment:
What's the purpose of this `0 + 1` here? Would it make sense to add comment here?

Reply:
Dependending on the `container.dumper.inline_class_loader`, the generated container can be one big file, or several ones nested in a subdirectory. This 0/1 is the corresponding offset (not-nested / nested).
Not sure a comment is needed, this is a generated file.

Reply:
Thanks, I see. Didn't consider that this is generated, but I guess then it's fine. At first I was thinking about some wired magic here :smile: